cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 10/1/2024 (PENDING)_
 **Bugfixes:**
 
 - Patched [find-process](https://github.com/yibn2008/find-process) to fix an issue where trying to clean up browser profiles can throw an error on Windows. Addresses [#30378](https://github.com/cypress-io/cypress/issues/30378).
+- Fixed an issue where requests to the same resource in rapid succession may not have the appropriate static response intercept applied if there are multiple intercepts that apply for that resource. 
```

Comment:
I think you need to link the PR or associated issue to make the semantic PR job happy

Reply:
Doh. I was in the process of adding this but forgot to push it.

Reply:
[`339a450` (#30435)](https://github.com/cypress-io/cypress/pull/30435/commits/339a450fe3ade676b440489e52971e0dd0706dd0)